tasty-query/shared/src/main/scala/tastyquery/reader/classfiles/ClassfileReader.scala

```diff
@@ -19,25 +19,33 @@ private[classfiles] final class ClassfileReader private () {
 
   transparent inline def pool(using pool: ConstantPool): pool.type = pool
 
-  def acceptHeader()(using DataStream): Unit = {
-    acceptMagicNumber()
-    acceptVersion()
+  def acceptHeader(classOwner: DeclaringSymbol, classRoot: ClassData)(using DataStream): Unit = {
```

Comment:
0.3 release?

Reply:
or I can remove these as they were only for improving the error debug message

Reply:
This class is `private[classfiles]`, so it's not part of the public API, so no need for 0.3. ;)